app/frontend/src/components/forms/submission/ManageSubmissionUsers.vue

```diff
@@ -292,7 +301,15 @@ export default {
       if (!input) return;
       this.isLoadingDropdown = true;
       try {
-        const response = await userService.getUsers({ search: input });
+        // The form's IDP (only support 1 at a time right now), blank is 'team' and should be IDIR
+        const idp =
+          this.form.identityProviders && this.form.identityProviders.length
+            ? this.form.identityProviders[0].code
+            : 'idir';
```

Comment:
Let's use the constants definitions if possible instead (utils/constants.js) -> `IdentityProviders.IDIR`

Reply:
👍